pkgs/by-name/in/instant-scala/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  scala-cli,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "instant-scala";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jatcwang";
+    repo = "instant-scala";
+    rev = "/refs/tags/v${finalAttrs.version}";
+    sha256 = "jqSvKTL8NzqjwqDj/+55YWecx2bnzuArP8RdfH5q/1U=";
+  };
+
+  buildInputs = [
+    scala-cli
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp instant-scala $out/bin/instant-scala
+    chmod +x $out/bin/instant-scala
+  '';
+
+  meta = {
+    description = "Write Scala scripts that starts instantly using scala-cli and GraalVM";
+    homepage = "https://github.com/jatcwang/instant-scala";
+    license = lib.licenses.asl20;
+    maintainers = [ lib.maintainer.jatcwang ];
+    mainProgram = "instant-scala";
+    platforms = lib.platforms.unix;
```

Comment:
```suggestion
```

Not needed, should be fine on all platforms that run Scala.

Reply:
removed